markdown/en/documentation/field-roles.md

```diff
@@ -0,0 +1,113 @@
+## FieldRoles
+
+This section describes an essential part of the integration
+of FACT-Finder using Web Components. FACT-Finder and Web Components _depend_
+on a proper definition of these roles. A missing definition can result in
+unexpected behaviors that might look like bugs.
+
+Certain fields/columns in FACT-Finder's product database can have special
+meanings. E.g. a _price_ field. Many of FACT-Finder's features, such as
+Tracking require knowledge of these special meanings. Since every customer
+might use custom _field names_ there needs to be a mapping between the
+_meaning_ of a field and its _field name_.
+We call these meanings `fieldRoles`. During the set-up process of a new
+channel in the FACT-Finder-UI you assign a field to each one of the
+`fieldRoles`.
+
+These are the `fieldRoles`:
+
+- `brand` – name of manufacturer.
+- `campaignProductNumber` – product number, that are used to reference the
+product in the campaign manager.
+- `deeplink` – URL of the product page.
+- `description` – product description.
+- `displayProductNumber` – The displayed product number.
+- `ean` – EAN
+- `imageUrl` – product image-URL.
+- `price` – price field.
+- `productName` – product name.
+- `trackingProductNumber` – product number used in tracking
+- `masterArticleNumber` – product number of the main product if product
+variants are used
+
+**IMPORTANT:** During set-up of a channel in the FACT-Finder-UI you have
+to assign each of the `fieldRoles` listed above to one field/column in
+your product database.
+
+This means in order for FACT-Finder and Web Components to process your
+data properly you also need to provide a mapping of `fieldRoles` to
+`field names` in your front-end before you request data via Web Components
+from any API except the search API.
+
+**In other words:** You might have a field called `prix` that contains the
+product's price in your product database. During set-up you assign the
+role `price` to your field `prix` in the FACT-Finder-UI. In order to use the [Currency Attributes](https://web-components.fact-finder.de/documentation/currency-guide) on your product detail page, you have to map `price` to
+`prix` in the `factfinder.communication.fieldRoles` property in your
+front-end.
+
+**NOTICE:** The `fieldRoles` are configured in the FACT-Finder-UI during
+the set-up process. It is essential the roles you define in the browser
+match the ones defined in FACT-Finder.
+
+#### How to define `fieldRoles`
```

Comment:
"... on the WebComponents side"
or similar sentence to clearly distinguish between FF UI field roles configuration and FF WC